src/sandstorm/run-bundle.c++

```diff
@@ -1183,18 +1189,17 @@ private:
     KJ_SYSCALL(sigprocmask(SIG_SETMASK, &sigset, nullptr));
   }
 
-  void copyEtc() {
+  void linkEtc() {
     auto files = splitLines(readAll("etc.list"));
```

Comment:
Presumably adding `/etc/resolvconf/run/resolv.conf` to `etc.list` will do the right thing, in that case.

(also uh did I just break my own Sandstorm install?)